types/chrome/index.d.ts

```diff
@@ -8357,6 +8357,14 @@ declare namespace chrome.sessions {
  * @since Chrome 20
  */
 declare namespace chrome.storage {
+    /**
+     * NoInfer for old TypeScript versions
+     * @internal
+     */
+    type NoInferX<T> = T[][T extends any ? 0 : never];
+    // The next line prevents things without the export keyword from being automatically exported (like NoInferX)
+    export {};
```

Comment:
Are any declarations being _lost_ with this at all? (i.e. any that are not explorted)

Reply:
No, because I added `export` for `AreaName` and `AccessLevel`, everything is accounted for.